addOns/ascanrules/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Fixed
 - Correct Context check in SQL Injection scan rule.
+- Java compatibility issues with Source Code Disclosure WEB-INF scan rule and newer JREs (Issue 4038).
```

Comment:
It needs to undo zaproxy/zap-extensions#1723. Probably better to do in its own PR or commit.

Reply:
Okay

Reply:
Split into: https://github.com/zaproxy/zap-extensions/pull/2959